src/libpriv/rpmostree-core.c

```diff
@@ -629,16 +617,13 @@ rpmostree_context_setup (RpmOstreeContext    *self,
                                    DNF_TRANSACTION_FLAG_NODOCS);
   }
 
-  { const char *const *ignore_scripts = NULL;
-    if (g_variant_dict_lookup (self->spec->dict, "ignore-scripts", "^a&s", &ignore_scripts))
-      {
-        g_autoptr(GHashTable) ignore_hash = NULL;
-
-        if (!rpmostree_script_ignore_hash_from_strv (ignore_scripts, &ignore_hash, error))
-          return FALSE;
-        rpmostree_context_set_ignore_scripts (self, ignore_hash);
-      }
-  }
+  /* We could likely delete this, but I'm keeping a log message just in case */
+  const char *const *ignore_scripts = NULL;
+  if (g_variant_dict_lookup (self->spec->dict, "ignore-scripts", "^a&s", &ignore_scripts))
```

Comment:
`g_variant_dict_contains` could also work here, and slightly simpler.  Unless an empty string array is likely enough that it warrants checking before the log message; that I don't know.